### PR TITLE
Timetable schema

### PR DESCRIPTION
**Pull request by Delt06**

### CODE REVIEW

File: db_schema/schema.md [hunk 1]
```diff
@@ -38,3 +38,13 @@ A record that indicates that a student is assigned to some sport activity.
 - **ActivityID** ```Integer, Foreign Key``` [(SportActivity.ID)](#sport-activity)
 
 ```Primary Key``` is the tuple (**StudentID**, **ActivityID**) since it is a many-to-many relationship.
+
+### Activity Schedule
+
+- **ActivityID** ```Integer, Foreign Key``` [(SportActivity.ID)](#sport-activity)
+- **Day** ```String(3)``` (day of the week in 3-letter format (e.g. "mon"))
```
Comment (illright) [inline db_schema/schema.md:45]: I'd say it's better to store the weekday as an integer from 0 to 6, sorting will be easier

File: db_schema/schema.md [hunk 2]
```diff
@@ -38,3 +38,13 @@ A record that indicates that a student is assigned to some sport activity.
...
+- **StartTime** ```Time, Not Null```
+- **FinishTime** ```Time, Not Null```
+- **Location** ```String(64), Not Null``` (where the training is conducted)
+
+```Primary Key``` is the tuple (**ActivityID**, **Day**).
```
Comment (illright) [inline db_schema/schema.md:50]: We should probably support several activities per day, so I'd suggest making a separate ID primary key

File: db_schema/schema.md (hunk 1 above)
Comment (Delt06) [reply db_schema/schema.md:45]: Accepted

File: db_schema/schema.md (hunk 2 above)
Comment (Delt06) [reply db_schema/schema.md:50]: Several schedule entries for the same activity in the same day?

Comment (illright) [reply db_schema/schema.md:50]: Yes

Comment (Delt06) [reply db_schema/schema.md:50]: Accepted

**Pull request merged**